apps/frontend/web/src/pages/Home.tsx

```diff
@@ -1,41 +1,61 @@
-import { Box, Button, Flex, Image, Text } from '@chakra-ui/react';
+import {
+  Box,
+  Button,
+  Center,
+  Flex,
+  Heading,
+  Image,
+  Text,
+} from '@chakra-ui/react';
 import { Link } from 'react-router-dom';
 
 const Home = () => {
   return (
-    <Box>
-      <Image
-        src="/overview.jpg"
-        h="100vh"
-        objectFit="cover"
-        alt="Overview"
-        zIndex={-1}
-        position="absolute"
-      />
-      <Flex justify="center" m="20">
-        <Box
-          w="50vw"
-          h="25vh"
-          bg="#797AA6"
-          opacity="90%"
-          border="1px"
-          borderRadius="5px"
-        >
-          <Text color="white" m="5px">
-            {'Welcome to DevManager!'}
+    <Box
+      bgImage="url('./overview.jpg')"
+      bgSize="100rem"
+      bgPosition="center"
+      bgRepeat="no-repeat"
+      w="100vw"
+      h="100vh"
+    >
+      <Center>
+        <Image src="./developer-splash.png" w="10%" />
+      </Center>
+      <Center>
+        <Heading color="#42B7B4" size="3xl">
+          {'Dev Manager'}
+        </Heading>
+      </Center>
+      <Center>
+        <Box w="500px" display="flex" align="center" mt="5rem" mb="5rem">
+          <Text color="black" m="5px">
+            {
+              'Welcome to DevManager! The first game that you can manage a full crew of a dev compagny! With a style 2D, and a gameplay simple and intuitive'
```

Comment:
'where you can'
pas de g a company ;)
'2d style', on inverse en anglais
pareil pour gameplay ;)